runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -215,7 +215,7 @@ else if (config.clue() && message.toLowerCase().startsWith("!clues") && message.
 			log.debug("Running clue lookup for {}", search);
 			executor.submit(() -> playerClueLookup(setMessage, search));
 		}
-		else if (message.toLowerCase().startsWith("!kc "))
+		else if (config.killcount() && message.toLowerCase().startsWith("!kc") && message.length() > 4)
```

Comment:
This should probably still be `.startsWith("!kc ")` despite it being inconsistent with the others as it is more-correct. Regardless it is out of scope of the pr.